crates/storage/errors/src/provider.rs

```diff
@@ -2,49 +2,44 @@ use reth_primitives::{
     Address, BlockHash, BlockHashOrNumber, BlockNumber, GotExpected, StaticFileSegment,
     TxHashOrNumber, TxNumber, B256, U256,
 };
+use derive_more::Display;
+use alloc::{boxed::Box, string::String};
+use crate::{db::DatabaseError, lockfile::StorageLockError, writer::UnifiedStorageWriterError};
 
 #[cfg(feature = "std")]
 use std::path::PathBuf;
 
-#[cfg(not(feature = "std"))]
-use alloc::{
-    boxed::Box,
-    string::{String, ToString},
-};
-
```

Comment:
please revert, when `std` feature is enabled, we can use the corresponding types from stdlib that are accessible without manual importing

Reply:
done